src/cfnlint/decode/cfn_json.py

```diff
@@ -537,16 +364,227 @@ class CfnJSONDecoder(json.JSONDecoder):
 
     def __init__(self, *args, **kwargs):
         json.JSONDecoder.__init__(self, *args, **kwargs)
-        self.parse_object = CfnJSONObject
+        self.parse_object = self.cfn_json_object
         self.parse_array = self.JSONArray
         self.parse_string = py_scanstring
         self.memo = {}
         self.object_pairs_hook = check_duplicates
         self.scan_once = py_make_scanner(self)
+        self.newline_indexes = []
+
+    def decode(self, s, _w=WHITESPACE.match):
+        """Overridden to retrieve indexes """
+        print(_w)
+        self.newline_indexes = find_indexes(s)
+        obj = json.JSONDecoder.decode(self, s)
+        return obj
 
     def JSONArray(self, s_and_end, scan_once, **kwargs):
         """ Convert JSON array to be a list_node object """
         values, end = json.decoder.JSONArray(s_and_end, scan_once, **kwargs)
         s, start = s_and_end
-        beg_mark, end_mark = get_beg_end_mark(s, start, end)
+        print(s)
```

Comment:
This needed to stay a tuple but would fail for s not being referenced. Need to better understand this logic